dashboard/config/routes.rb

```diff
@@ -198,6 +198,7 @@
         get "/#{key}/:channel_id/edit", to: 'projects#edit', key: key.to_s, as: "#{key}_project_edit"
         get "/#{key}/:channel_id/view", to: 'projects#show', key: key.to_s, as: "#{key}_project_view", readonly: true
         get "/#{key}/:channel_id/embed", to: 'projects#show', key: key.to_s, as: "#{key}_project_iframe_embed", iframe_embed: true
+        get "/#{key}/:channel_id/embed_app_and_code", to: 'projects#show', key: key.to_s, as: "#{key}_project_iframe_embed_app_and_code", iframe_embed_app_and_code: true, readonly: true
```

Comment:
how is embed_app_and_code different than embed? sort of the same question as my other comments

Reply:
embed returns a view that only has the app itself. embed_app_and_code has that, and the code.

I think maybe if 'embed' were renamed to 'embed_app', the distinction would be more clear?

Reply:
ah, yes, that would be clearer!

If you do rename this, some extra steps will be needed to avoid errors around the time of the deploy. specifically, I would recommend leaving the old `embed` route up for a couple days, so that clients which have old js code pointing to the old route, can still access it. probably this is worth doing in a separate PR.

Reply:
Sounds good! Thanks for pointing that out - I made a follow up jira item to address this, to do after this ships to avoid taking on tech debt: https://codedotorg.atlassian.net/browse/LP-1455